Makefile

```diff
@@ -138,19 +138,23 @@ build-ui: clean-ui ## build the Argo CD UI extension creating the ui/extension.t
 	yarn --cwd ${UI_DIR} install
 	yarn --cwd ${UI_DIR} build
 
-.PHONY: update-openapi-ui
-update-openapi-ui:  build goreman  ## Update the OpenAPI spec from the local server.
+.PHONY: openapi-ui
+openapi-ui:  build-go goreman  ## Update the OpenAPI spec from the local server.
+	@echo "Updating OpenAPI specification..."
 	@echo "Starting the backend server"
 	goreman start backend &
 	sleep 5
 	@echo "Downloading OpenAPI spec from dev server"
 	yarn --cwd ${UI_DIR} api:download
 	killall goreman || true
 
-.PHONY: codegen-ui
-codegen-ui: ## Generate the UI API files based on the OpenAPI spec.
-	yarn --cwd ${UI_DIR}  api:generate
 
+.PHONY: codegen-ui openapi-ui
+codegen-ui: openapi-ui ## Generate the UI API files based on the OpenAPI spec.
+	@echo "Generating API files..."
+	UI_DIR=${CURRENT_DIR}/ui
+	yarn --cwd ${UI_DIR} api:generate
+	rm -f ${UI_DIR}/src/gen/schema.yaml
```

Comment:
Are you sure that we should remove this file as the last step?

Reply:
I want to remove the schema.yaml, since it would be create to store the schema which is used to  generate the types,  we don't need this file in the repo.